src/UI/CLI/Bus/ConsumeEvents.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\UI\CLI\Bus;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ConsumeEvents extends Command
+{
+    public function __construct()
+    {
+        parent::__construct('pollhq:bus:consume:events');
+    }
+
+
+    protected function execute(InputInterface $input, OutputInterface $output) : void
+    {
+        $output->writeln('Consuming events.');
```

Comment:
Events are currently based through the event bus in the same process as the command handling however they could be consume in a separate process.